src/components/data/aboutData.js

```diff
@@ -0,0 +1,24 @@
+const aboutData = [
+  {
+    id: 1,
+    slug: 'about-app',
+    title: 'About the App',
+    description:
+      'In this app, you can add, delete, submit and edit items. To edit items, simply double click on it. Once you are done, press the enter key to resubmit. This app will persist your data in the browser local storage. So whether you reload, close your app or reopened it, you still have access to your to-dos items.',
+  },
+  {
+    id: 2,
+    slug: 'about-author',
+    title: 'About the Author',
```

Comment:
It's recommended to add a **title** to the project for better **SEO** and **user experience**,
this can be achieved either through your `Html` or `packacge.json` files    Please add a descriptive title to the project.